### PR TITLE
ACP-10 C3

### PR DESCRIPTION
**Pull request by OlanoB**
Head of Household
Parent's Name
Parent's Phone number

### CODE REVIEW

File: templates/C/3.html [hunk 1]
```diff
@@ -0,0 +1,19 @@
+<div id="C3">
+<p>
+<label><b>Head of Household:<br></b>
+<input type="text" Head_Of_Household="Head_Of_Household" required>
```
Comment (xispa) [inline templates/C/3.html]: As per other code I've seen from @austinsdoe , if you add the attribute "required" in an input field, you should also guarantee consistency by adding the class "required" in the label (so an asterisk will be displayed after the label). See https://github.com/naralabs/accel.idsr/blob/wip/static/css/main.css#L303

Comment (xispa) [inline templates/C/3.html]: Also, something weird here. Should look like:

`<input type="text" id="head_of_household" name="head_of_household" required>`

note this is the way how the id and name should be set in a "input" element (and in fact, in any other HTML element you may think of)

File: templates/C/3.html [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+</label>
+</p>
+
+<p>
+<label><b>Parent's Name (if young):<br></b>
+<input type="text" Parent_Name="Parents_Name" required>
```
Comment (xispa) [inline templates/C/3.html]: Same as https://github.com/naralabs/accel.idsr/pull/11/files#r86813395
See https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input

File: templates/C/3.html [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+</label>
+</p>
+
+<p>
+<label><b>Parent's Phone Number:<br></b>
+<input type="tel" Parent's_Phone_Number" maxlength="10" required>
```
Comment (xispa) [inline templates/C/3.html]: Same as https://github.com/naralabs/accel.idsr/pull/11/files#r86813395
See https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input

File: templates/C/3.html [hunk 4]
```diff
@@ -0,0 +1,19 @@
+<div id="C3">
+<p>
+<label><b>Head of Household:<br></b>
+<input type="text" Head_Of_Household="Head_Of_Household" required>
+</label>
```
Comment (OlanoB) [inline templates/C/3.html]: OK, thanks
i will make the changes or corrections and get back ASAP

File: templates/C/3.html (hunk 1 above)
Comment (OlanoB) [inline templates/C/3.html]: Oh!, Ok Thanks, 
quick rectification. I will work on it

File: templates/D/2.html [hunk 5]
```diff
@@ -0,0 +1,25 @@
+<div id="D2">
+<form>
+<div>
+<p>
+<label for="name_of_person_reporting" class="required">Name of Person Reporting:</label><br>
+<input type="text" id="name_of_person_reporting" name="name_of_person_reporting" required>
+</p>
+
+<p>
+<label for="phone_number_of_person_reporting" class=required">Phone Number of Person Reporting:</label><br>
```
Comment (xispa) [inline templates/D/2.html]: should be `class="required"` (note there is a quotation missing)

Comment (OlanoB) [inline templates/D/2.html]: Ok, will be up a few minutes

Comment (OlanoB) [inline templates/D/2.html]: The code for ACP-13 D2 seem to have it right. NOTE: i did some PRs today, so please check
the PR you send might be for yesterday. I have rectified some common mistakes today.

Comment (OlanoB) [inline templates/D/2.html]: any update?

**Pull request merged**